### PR TITLE
Onboarding cloud-bulldozer/e2e-benchmarking to PROW

### PR DESCRIPTION
**Pull request by vishnuchalla**
Onboarding cloud-bulldozer/e2e-benchmarking to PROW

### CODE REVIEW

**Comment by vishnuchalla** [comment]
pre-requisite PR: https://github.com/cloud-bulldozer/e2e-benchmarking/pull/673

**Comment by vishnuchalla** [comment]
@jtaleric I believe we have already installed openshift ci and merger bots into our repo. Can you please confirm if we are missing something here: https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#granting-robots-privileges-and-installing-the-github-app? One of the checks is failing due to access issues.

File: ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: echo "test"
```
Comment (jtaleric) [inline ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml:28]: This a WIP PR?

Comment (vishnuchalla) [reply ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml:28]: No its a actual PR. I want e2e onboarded to PROW first, verify its working in e2e and then take tests in a different PR.

**Review by jtaleric — APPROVED** [review]
/lgtm

**Pull request merged**